### ISSUE TITLE
Unable to read from registry/keys for Windows 10  x64 dumps

### ISSUE THREAD

**Issue by ATrashInTheWorld** [volatilityfoundation/volatility#488]
Sorry to ask, I'm new to GitHub.

I'm actually having the same problem as in the following link:
https://github.com/volatilityfoundation/volatility/issues/104

The issue is that there is no result or error from any hive commands.

ex;
```
volatility -f DUMPFILE.dmp --profile=Win10x64_15063 hivelist
Volatility Foundation Volatility Framework 2.6
Virtual            Physical           Name
------------------ ------------------ ----

```
I-m actually working on a Windows 10 x64  version 1709 OS build machine

`Here are more specification about my machine. Tell me if you need more:`
```
`      Suggested Profile(s) : Win10x64_10586, Win10x64_14393, Win10x64, Win2016x64_14393, Win10x64_15063 (Instantiated with Win10x64_15063)
                     AS Layer1 : SkipDuplicatesAMD64PagedMemory (Kernel AS)
                     AS Layer2 : WindowsCrashDumpSpace64 (Unnamed AS)
                     AS Layer3 : FileAddressSpace (/root/lol/ANALYSIS_FORENSICS/BOGANEL-20180114-004634.dmp)
                      PAE type : No PAE
                           DTB : 0x1ab000L
                          KDBG : 0xf801d65ee4d0L
          Number of Processors : 4
     Image Type (Service Pack) : 0
                KPCR for CPU 0 : 0xfffff801d4e66000L
                KPCR for CPU 1 : 0xffffb681e4e20000L
                KPCR for CPU 2 : 0xffffb681e4ec0000L
                KPCR for CPU 3 : 0xffffb681e4f4d000L
             KUSER_SHARED_DATA : 0xfffff78000000000L
           Image date and time : 2018-01-14 00:47:05 UTC+0000
     Image local date and time : 2018-01-13 19:47:05 -0500


`
volatility -f DUMPFILE.dmp --profile=Win10x64_15063 kdbgscan
Volatility Foundation Volatility Framework 2.6
**************************************************
Instantiating KDBG using: Unnamed AS Win10x64_15063 (6.4.15063 64bit)
Offset (V)                    : 0xf801d65ee4d0
Offset (P)                    : 0x228bee4d0
KdCopyDataBlock (V)           : 0xf801d649c9c4
Block encoded                 : Yes
Wait never                    : 0xaf3e333b00b90bc3
Wait always                   : 0x5c85cd77419ad800
KDBG owner tag check          : True
Profile suggestion (KDBGHeader): Win10x64_15063
Version64                     : 0xf801d65f1678 (Major: 15, Minor: 16299)
Service Pack (CmNtCSDVersion) : 0
Build string (NtBuildLab)     : 16299.15.amd64fre.rs3_release.17
PsActiveProcessHead           : 0xfffff801d65fdaa0 (215 processes)
PsLoadedModuleList            : 0xfffff801d6604fd0 (218 modules)
KernelBase                    : 0xfffff801d629e000 (Matches MZ: True)
Major (OptionalHeader)        : 10
Minor (OptionalHeader)        : 0
KPCR                          : 0xfffff801d4e66000 (CPU 0)
KPCR                          : 0xffffb681e4e20000 (CPU 1)
KPCR                          : 0xffffb681e4ec0000 (CPU 2)
KPCR                          : 0xffffb681e4f4d000 (CPU 3)

```

However, I just realized that the command worked on another memory scan from another Windows 10 computer. But again I could find the \REGISTRY\MACHINE\SYSTEM from the second computer.

If the issue has been solved, please redirect me toward. I've searched for some answers, but didn't really find any answers. Again, If this issue is solved somewhere please tell me links.
Thank you for your time.

**Comment by ATrashInTheWorld** [comment]
I'm not sure about the answer, but the way I use volatility is within Kali Linux, which is already "install".
I've also, multiple time, updated and upgraded my Linux version.

**Comment by Beercow** [comment]
Same issue here. Tried creating the vtypes from ntoskrnl.exe but the results were the same. So it doesn't appear to be as easy of a fix like in #104.

Volatility Foundation Volatility Framework 2.6
**************************************************
Instantiating KDBG using: Unnamed AS Win10x64_15063 (6.4.15063 64bit)
Offset (V)                    : 0xf803c4dbe4f0
Offset (P)                    : 0x229bbe4f0
KdCopyDataBlock (V)           : 0xf803c4c7dfbc
Block encoded                 : Yes
Wait never                    : 0x4f35fce4023a76ae
Wait always                   : 0x474ef4525bfa98
KDBG owner tag check          : True
Profile suggestion (KDBGHeader): Win10x64_15063
Version64                     : 0xf803c4dc10a0 (Major: 15, Minor: 15063)
Service Pack (CmNtCSDVersion) : 0
Build string (NtBuildLab)     : 15063.0.amd64fre.rs2_release.170
PsActiveProcessHead           : 0xfffff803c4dcc120 (213 processes)
PsLoadedModuleList            : 0xfffff803c4dd2720 (206 modules)
KernelBase                    : 0xfffff803c4a7f000 (Matches MZ: True)
Major (OptionalHeader)        : 10
Minor (OptionalHeader)        : 0
KPCR                          : 0xfffff803c3305000 (CPU 0)
KPCR                          : 0xffffab01c4521000 (CPU 1)
KPCR                          : 0xffffab01c4568000 (CPU 2)
KPCR                          : 0xffffab01c4640000 (CPU 3)

**Comment by ratarun** [comment]
I faced a similar issue with a Win10 x64 image. It turned out I had chosen the wrong profile. Using the profile Win10x64_17763 gave me a blank output for `hivelist`, but using the profile Win10x64_15063 gave me the required result. The list of supported profiles can be obtained by `python vol.py --info`. Try out the many profiles listed for Win10 x64. One may work
